# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../ffmpeg/gcc34.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Due to gcc 3.4 optimizations some constants are optimized away - so
skip MMX optimizations for now ...

  - Rene Rebe <rene@exactcode.de>

--- ffmpeg-0.4.9-pre1/libavcodec/liba52/resample.c.orig	2003-11-03 19:06:54.000000000 +0100
+++ ffmpeg-0.4.9-pre1/libavcodec/liba52/resample.c	2004-09-15 13:17:02.079238694 +0200
@@ -13,14 +13,14 @@
 
 #include "resample_c.c"
 
-#ifdef ARCH_X86
+#ifdef HAVEMMX_ARCH_X86
 #include "resample_mmx.c"
 #endif
 
 void* a52_resample_init(uint32_t mm_accel,int flags,int chans){
 void* tmp;
 
-#ifdef ARCH_X86
+#ifdef HAVEMMX_ARCH_X86
     if(mm_accel&MM_ACCEL_X86_MMX){
 	tmp=a52_resample_MMX(flags,chans);
 	if(tmp){
